.github/workflows/storybook.yml

```
@@ -19,6 +19,7 @@ jobs:
         run: |
           npm ci
           npm run build-storybook-ci
+          touch .nojekyll
```

```suggestion
          touch ./docs-build/.nojekyll
```
might place the file in the build folder?